### DISCUSSION TITLE
After updating Neovim to v0.10, the colors of the completion menu have changed

### DISCUSSION THREAD

**Discussion by optic0n** [neoclide/coc.nvim#5140] [Q&A]
![image](https://github.com/user-attachments/assets/1e0d5d16-35b3-4b78-8c8b-ad0349ec2fd6)
![image](https://github.com/user-attachments/assets/c9ac0eb0-8aef-458c-9f95-41085bc7578d)

"After updating Neovim to version 0.10.0, the colors of the completion menu changed, as shown in the screenshots. In the left part of screenshot, I'm using Arch Linux with NVIM v0.10.0, and in the right, I'm using Ubuntu with NVIM v0.7.2. The color of the autocomplete menu differs between versions, and it doesn't seem to be related to the colorscheme. The issue appears to be with `coc.nvim`, as removing this plugin restores the previous colors in the completion menu. Any thoughts?

**Comment by chemzqm** [comment]
Highlights used by neovim treesister have changed which linked by semantic highlight groups of coc.nvim, you can overwrite them see `:h CocSem`

**Reply by optic0n** [reply]
Disabling `g:coc_default_semantic_highlight_groups` (let g:coc_default_semantic_highlight_groups = 0) didn't work. Are you suggesting manually overriding the highlights? For example:

```
" Fix Coc.nvim completion menu colors
highlight Pmenu guibg=#1e1e1e guifg=#d4d4d4
highlight PmenuSel guibg=#5c6370 guifg=#ffffff
highlight PmenuSbar guibg=#2c2c2c
highlight PmenuThumb guibg=#5c6370

" Fix Coc.nvim semantic highlighting
highlight link CocSemType Type
highlight link CocSemTypeModClassDeclaration ClassDeclaration
highlight link CocSemFunction Function
highlight link CocSemVariable Variable

```

**Reply by chemzqm** [reply]
See `:h CocPum` for popup menu related highlight groups.

**Reply by optic0n** [reply]
So, do I need to manually assign the colors? For example: `highlight CocPum guifg=#c6c8d1 guibg=#3d425b`. Is there no command to simply revert to the default colors? Apologies if I'm misunderstanding, as I'm new to Neovim. Also, I'm curious why does no one else seem to have this issue? I can share my config if you'd like to take a look.

**Reply by chemzqm** [reply]
Yes, you can use link too, see `:hi`
